src/Files.App/Commands/HotKeyHelpers.cs

```diff
@@ -1,13 +1,146 @@
+			[VirtualKey.Home] = "Home".GetString(),
+			[VirtualKey.End] = "End".GetString(),
+			[VirtualKey.PageDown] = "PageDown".GetString(),
+			[VirtualKey.PageUp] = "PageUp".GetString(),
+			[VirtualKey.Separator] = "Separator".GetString(),
+			[VirtualKey.Pause] = "Pause".GetString(),
+			[VirtualKey.Sleep] = "Sleep".GetString(),
+			[VirtualKey.Clear] = "Clear".GetString(),
+			[VirtualKey.Print] = "Print".GetString(),
+			[VirtualKey.Help] = "Help".GetString(),
+			[VirtualKey.XButton1] = "Mouse4".GetString(),
+			[VirtualKey.XButton2] = "Mouse5".GetString(),
+			[VirtualKey.F1] = "F1",
+			[VirtualKey.F2] = "F2",
+			[VirtualKey.F3] = "F3",
+			[VirtualKey.F4] = "F4",
+			[VirtualKey.F5] = "F5",
+			[VirtualKey.F6] = "F6",
+			[VirtualKey.F7] = "F7",
+			[VirtualKey.F8] = "F8",
+			[VirtualKey.F9] = "F9",
+			[VirtualKey.F10] = "F10",
+			[VirtualKey.F11] = "F11",
+			[VirtualKey.F12] = "F12",
+			[VirtualKey.F13] = "F13",
+			[VirtualKey.F14] = "F14",
+			[VirtualKey.F15] = "F15",
+			[VirtualKey.F16] = "F16",
+			[VirtualKey.F17] = "F17",
+			[VirtualKey.F18] = "F18",
+			[VirtualKey.F19] = "F19",
+			[VirtualKey.F20] = "F20",
+			[VirtualKey.F21] = "F21",
+			[VirtualKey.F22] = "F22",
+			[VirtualKey.F23] = "F23",
+			[VirtualKey.F24] = "F24",
+			[VirtualKey.Number0] = "0",
+			[VirtualKey.Number1] = "1",
+			[VirtualKey.Number2] = "2",
+			[VirtualKey.Number3] = "3",
+			[VirtualKey.Number4] = "4",
+			[VirtualKey.Number5] = "5",
+			[VirtualKey.Number6] = "6",
+			[VirtualKey.Number7] = "7",
+			[VirtualKey.Number8] = "8",
+			[VirtualKey.Number9] = "9",
+			[VirtualKey.NumberPad0] = "Pad0".GetString(),
+			[VirtualKey.NumberPad1] = "Pad1".GetString(),
+			[VirtualKey.NumberPad2] = "Pad2".GetString(),
+			[VirtualKey.NumberPad3] = "Pad3".GetString(),
+			[VirtualKey.NumberPad4] = "Pad4".GetString(),
+			[VirtualKey.NumberPad5] = "Pad5".GetString(),
+			[VirtualKey.NumberPad6] = "Pad6".GetString(),
+			[VirtualKey.NumberPad7] = "Pad7".GetString(),
+			[VirtualKey.NumberPad8] = "Pad8".GetString(),
+			[VirtualKey.NumberPad9] = "Pad9".GetString(),
+			[VirtualKey.A] = "A",
+			[VirtualKey.B] = "B",
+			[VirtualKey.C] = "C",
+			[VirtualKey.D] = "D",
+			[VirtualKey.E] = "E",
+			[VirtualKey.F] = "F",
+			[VirtualKey.G] = "G",
+			[VirtualKey.H] = "H",
+			[VirtualKey.I] = "I",
+			[VirtualKey.J] = "J",
+			[VirtualKey.K] = "K",
+			[VirtualKey.L] = "L",
+			[VirtualKey.M] = "M",
+			[VirtualKey.N] = "N",
+			[VirtualKey.O] = "O",
+			[VirtualKey.P] = "P",
+			[VirtualKey.Q] = "Q",
+			[VirtualKey.R] = "R",
+			[VirtualKey.S] = "S",
+			[VirtualKey.T] = "T",
+			[VirtualKey.U] = "U",
+			[VirtualKey.V] = "V",
+			[VirtualKey.W] = "W",
+			[VirtualKey.X] = "X",
+			[VirtualKey.Y] = "Y",
+			[VirtualKey.Z] = "Z",
+			[VirtualKey.Add] = "+",
+			[VirtualKey.Subtract] = "-",
+			[VirtualKey.Multiply] = "*",
+			[VirtualKey.Divide] = "/",
+			[(VirtualKey)188] = ",",
+			[(VirtualKey)190] = ".",
+			[(VirtualKey)192] = "`",
```

Comment:
Shouldn't we localize them as well?
In Italian `(VirtualKey)192` is `ò`, it would make more sense if that was the key displayed

Reply:
@ferrariofilippo @yaira2 These particular keys (oem) must not be localized because they depend on the user layout and not on his language. You can have an Italian keyboard and use Windows and Files in English for example. I found a piece of code that returns the key used for these oems depending on the layout used. I will update the code.

Reply:
General guidance from Microsoft is not to translate hot keys. We don't have to follow this guidance but something to keep in mind.

Reply:
@ferrariofilippo It's done, You should see ò now. It is not displayed, it will be necessary to display Commands.OpenTerminal.LabelWithHotkey somewhere in the application to test.

Reply:
I'll test it asap

Reply:
> @ferrariofilippo It's done, You should see ò now. It is not displayed, it will be necessary to display Commands.OpenTerminal.LabelWithHotkey somewhere in the application to test.

Everything works fine. Thank you for your work 👍